SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -40,6 +40,11 @@ void ASpatialMetricsDisplay::BeginPlay()
 
 	WorkerStats.Reserve(PreallocatedWorkerCount);
 	WorkerStatsLastUpdateTime.Reserve(PreallocatedWorkerCount);
+
+	if (!GetWorld()->IsServer() && GetDefault<USpatialGDKSettings>()->bEnableMetricsDisplay)
+	{
+		ToggleStatDisplay();
```

Comment:
Since `ToggleStatDisplay` is blueprint callable doesn't this mean we should give the user the choice of when to turn this on rather than in BeginPlay? Not super important though.

Reply:
The user has no easy means of getting access to this actor. They'd have to search for all actors of type or some such. The reason behind the flag is to enable the display though, so I think it's safe to default it to on.